README.md

```diff
@@ -3,9 +3,9 @@ GoPro Metadata Format Parser
 
 TLDR:
 
-1. `ffmpeg -y -i GOPR0001.MP4 -codec copy -map 0:m:handler_name:"	GoPro MET" -f rawvideo GOPR0001.bin`
-2. `gopro2json -i GOPR0001.bin -o GOPR0001.json`
-3. There is no step 3
+1. `ffmpeg -y -i GOPR0001.MP4 -codec copy -map 0:3 -f rawvideo GOPR0001.bin`
```

Comment:
Exactly, and I merged that change :) I had a hard time getting the tab character to work and be copy/paste friendly. Silly how GoPro put a tab character in the handler name!